docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -40,7 +40,7 @@ export default function AppFrameBanner() {
           }),
       ]}
     >
-      🚀 MUI X v6 is out! Discover what&apos;s new and get started now!
+      🚀 The MUI Developer Survey 2023 is live! Participate and help shape MUI's roadmap for 2024!
```

Comment:
```suggestion
      🚀 Influence MUI's roadmap by answering the 2023 developer survey!
```

I think this one should be as short as possible — here's a possibility, so we don't repeat "MUI" and don't mention both 2023 and 2024.

Reply:
Called the same point as Sam [on Slack](https://mui-org.slack.com/archives/C02NLU6DVEF/p1704713886830519), too — I'm up for either making it a "retrospective" survey (which I don't think it is; it's forward-thinking) or turning the page and committing to being the 2024 dev survey. 😬